airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json

```diff
@@ -67,6 +67,24 @@
     "segments.date": {
       "type": ["null", "string"],
       "format": "date"
+    },
+    "campaign.bidding_strategy_type": {
+      "type": ["null", "string"]
+    },
+    "metrics.clicks": {
+      "type": ["null", "integer"]
+    },
+    "metrics.cost_micros": {
+      "type": ["null", "integer"]
+    },
+    "metrics.impressions": {
+      "type": ["null", "integer"]
+    },
+    "ad_group_criterion.criterion_id": {
+      "type": ["null", "integer"]
+    },
+    "segments.ad_network_type": {
```

Comment:
@schlattk  Error happens because this field was added.

message: \"Cannot select the following segments because at least one unsupported metric is found in SELECT or WHERE clause: \\'segments.ad_network_type\\'(unsupported metrics: \\'historical_quality_score\\').\"\n}\nrequest_id: \"hBx3G90_bSZ-rVpnBMQlxQ\"\n, 'hBx3G90_bSZ-rVpnBMQlxQ')"}}

Reply:
It works when `read` command is run without this field.

Reply:
@augan-rymkhan ok but it is a supported field according to documentation and also it has been running live for a long time without any errors. What do you suggest is the solution?

Reply:
@schlattk It seems we can not query  `metrics.historical_quality_score` and  `segments.ad_network_type` together.

In docs:
_A segment in the SELECT clause is incompatible with a metric in the SELECT or WHERE clause._
For more you can read [here](https://developers.google.com/google-ads/api/reference/rpc/v7/QueryErrorEnum.QueryError#prohibited_segment_with_metric_in_select_or_where_clause).

Reply:
yes @augan-rymkhan that appears to be the problem. Ok, I removed the field I think we can most probably do without it in this report.